src/content/en/showcase/2017/santa.md

```diff
@@ -0,0 +1,217 @@
+
+<figure>
+  <img src="/web/showcase/2017/images/santa/cache-manifest.png"  />
+</figure>
+
+In Santa Tracker, upgrading the Service Worker also forces an immediate refresh of our clients—there's not too much state in a Santa game that a user risks losing—although this strategy is up to you.
+
+### Offline Browsing Experience
+
+Of course, we also had to make some changes to the UI to support an offline experience—and to make it easier to understand for users who might not expect that a website can work offline.
+
+A small banner tells you when you're browsing offline.
+All scenes that aren't cached are "frozen" and unclickable.
+That way, users aren't able to access content that isn't available.
+
+<figure class="attempt-right">
+  <img src="/web/showcase/2017/images/santa/offline.png"  />
+</figure>
+
+Santa Tracker makes regular requests to Santa's API.
+If these requests fail or timeout, we assume that the user is offline.
+We use this API rather than the browser's built-in [`navigator.onLine` property](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine): this would only inform us whether the user is maybe online (this is also known as Lie-Fi).
+
+### The International Connection
+
+While the majority of our users are in English—followed by Japanese, Portuguese, Spanish and French—Santa is built and released in over 35 different languages.
+
+When a user loads Santa Tracker, we use the [browser's language](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) and other cues to choose a language to serve.
+The majority of users never overwrite this language.
+However, if a user chooses a new language through our picker, we treat this as if an upgrade is available—just as in the case above, when a new version of Santa Tracker is available.
+
+<figure>
+  <img src="/web/showcase/2017/images/santa/language.gif"  />
+</figure>
+
+To put it another way, the current version of Santa Tracker for the purposes of the Service Worker, is actually a tuple of _(build,language)_.
+
+## Add To Home Screen
+
+Because Santa works offline and provides a Service Worker, eligible users are prompted to install it to their home screen.
+In 2016, around 10% of eligible loads were from this home screen icon.
+
+## Conclusion
+
+We were able to rapidly convert Santa Tracker into an offline PWA—enabling a reliable, engaging experience—thanks to our existing scene design, made easy through our existing use of Polymer and Web Components.
+It also leverages our build system to perform efficient upgrades, invalidating changed assets only.
+
+While Santa is largely a custom-built solution, many of its principles can be found in the Polymer Project's [App Toolbox](https://www.polymer-project.org/1.0/toolbox/).
+We suggest you check it out if you're building a new PWA from scratch—or, if you're looking for a framework-agnostic approach, try the the [Service Worker Helpers Library](https://github.com/GoogleChrome/sw-helpers).
```

Comment:
I don't think we're ready to start talking about sw-helpers yet. 

@addyosmani ?